src/models/platform.ts

```diff
@@ -304,6 +306,7 @@ export class PlatformInformation {
             case 'rhel':
                 return Runtime.RHEL_7;
             case 'debian':
+            case 'deepin':
                 return Runtime.Debian_8;
```

Comment:
Since this returns `Runtime.Debian_8` for `deepin` here, does the runtimes enum need to include `Runtime.deepin` above on line 30?

Reply:
Yes, it's true, it would not be necessary.

Reply:
I re-run the extension installer with this commit and everything works:
![deepinscreenshot_code_20180601200610](https://user-images.githubusercontent.com/19435185/40867009-5fb000d2-65d7-11e8-9c03-ccfb781996f7.png)

Reply:
Great, thank you!